packages/shell-api/src/collection.ts

```diff
@@ -350,8 +353,11 @@ export default class Collection extends ShellApiWithMongoClass {
    *
    * @returns {Array} The promise of the result.
    */
+  async distinct(field: string): Promise<Document>
+  async distinct(field: string, query: Document): Promise<Document>
+  async distinct(field: string, query: Document, options: DistinctOptions): Promise<Document>
   @returnsPromise
-  async distinct(field: string, query: Document, options: DistinctOptions = {}): Promise<Document> {
+  async distinct(field: string, query?: Document | undefined, options: DistinctOptions = {}): Promise<Document> {
```

Comment:
```suggestion
  async distinct(field: string, query?: Document, options: DistinctOptions = {}): Promise<Document> {
```

because ? already implies `undefined`?

Reply:
Yup, true... 👍